cypress/e2e/cot/main.py

```diff
@@ -2,10 +2,8 @@
 
 
 @cl.on_message
-async def main():
-    root_message = cl.user_session.get("root_message")  # type: cl.Message
-
-    tool1_msg = cl.Message(content="", author="Tool 1", parent_id=root_message.id)
+async def main(message: str, message_id: str):
+    tool1_msg = cl.Message(content="", author="Tool 1", parent_id=message_id)
```

Comment:
Why not pass a real message object instead? (We'd then use `message.content` and `message.id`)